sortLoot.py

```diff
@@ -8,41 +8,54 @@ def PlotRisk(): # goes through enemies potential attacks and damage according to
 def PlotDamage(): # goes through potential attacks and their damage to the enemy depending on his defense profile
     pass
    
-def SortLoot(loot): # sort the loot array into our different bags
-    weapons = []
-    defense = []
-    food = []
-    supplies = []
+        
     
-    for item in loot:
-        if('weapon' in item):
-            item.remove('weapon')
-            weapons.append([item[0]] + item[1])
+# lootDict = {'item1' : {'itemName' : 'MagicWand','itemType' : 'weapon', 'itemStats': [3, 2, 1] },
             
-        elif('defense' in item):
-            item.remove('defense')
-            defense.append([item[0]] + item[1])
-        elif('food' in item):
-            item.remove('food')
-            food.append([item[0]] + item[1])
-        elif('supplies' in item):
-            item.remove('supplies')
-            supplies.append(item)
-        
+#             'item2' : {'itemName' : 'gloves', 'itemType' : 'defense','itemStats': [1, 1, 2] }
+#             }
+
+playerInventory = {}
+
+lootDict = {'MagicWand': ['weapon', 3, 2, 1],
+             'gloves' : ['defense', 1, 1, 2],
+             'banana' : ['food', 10, 5],
+             'tape' : ['supplies', 5]}
     
-  
+         
+def SortLoot(loot: dict, playerInv: dict):

```

Comment:
Remove unused `playerInv` parameter. Other than that it has my approval.

Reply:
If you're feeling nice you can add a return type hint
`def SortLoot(loot: dict) -> dict:`